.travis.yml

```diff
@@ -58,5 +58,7 @@ script:
   # in the makefile
   - if [ $REPO_PROJ == "fvp" ]; then mkdir -p $HOME/$REPO_PROJ/Foundation_Platformpkg; fi
   - cd $HOME/$REPO_PROJ && repo init -u https://github.com/OP-TEE/manifest.git -m $REPO_PROJ.xml </dev/null && repo sync -j2 --no-clone-bundle --no-tags --quiet
+  # Fetch a local copy of dtc+libfdt to avoid issues with a possibly outdated libfdt-dev
+  - if [ $REPO_PROJ == "qemu_v8" ]; then cd $HOME/$REPO_PROJ/qemu && git submodule update --init dtc; fi
```

Comment:
@jforissier
`--recursive` is performed not only in the submodules of the current repo, but also in any nested submodules inside those submodules (and so on). I haven't checked if pixman or dtc has their own submodules, but that's why I prefer to use `--recursive` just to avoid checking this :)

But, anyway, I totally agree with other points, so generic solution is redundant

`Reviewed-by: Igor Opaniuk <igor.opaniuk@linaro.org>`